### DISCUSSION TITLE
Panda CSS with Astro, example of importing index.css in root component

### DISCUSSION THREAD

**Discussion by Magnus2142** [chakra-ui/panda#2794] [Ideas]
### Subject

Panda CSS + Astro

### Description

In the (docs for settings up Panda CSS in a Astro project)[https://panda-css.com/docs/installation/astro], it would be nice to make it more clear that you have to import the index.css in your root component in the project.

At step 4 you're told to add a `index.css` file in the src folder, and then import it in the root component of your project.
![image](https://github.com/user-attachments/assets/19117c1e-1cea-41c5-b5f5-7ded316e3c88)

It is pretty obvious now that I realised my mistake, but I suspect others may overlook this instruction as well. Adding a code block underneath the first one, showing an example of importing the `index.css` into a `src/pages/index.astro` file could be helpful. 😄

**Comment by segunadebayo** [comment]
Good suggestion. I've added this to the docs.
